keras_cv/layers/object_detection/nms_prediction_decoder.py

```diff
@@ -19,7 +19,7 @@
 
 # TODO(tanzhenyu): provide a TPU compatible NMS decoder.
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
-class NmsDecoder(tf.keras.layers.Layer):
+class NonMaxSuppression(tf.keras.layers.Layer):
```

Comment:
I'm fine with either NMS or NonMaxSuppression., but I prefer it be more specific, for example, MultiClassNMS or BatchedNMS.

Reply:
Sounds good - I'll think about a more descriptive name.

Reply:
Updated to MultiClassNMS